presto-resource-group-managers/src/test/java/com/facebook/presto/resourceGroups/TestStaticSelector.java

```diff
@@ -28,39 +32,171 @@
+    }
+
+    @Test
+    public void testResourceEstimatesRange()
+    {
+        ResourceGroupId resourceGroupId = new ResourceGroupId(new ResourceGroupId("global"), "foo");
+
+        StaticSelector smallQuerySelector = new StaticSelector(
+                Optional.empty(),
+                Optional.empty(),
+                Optional.empty(),
+                Optional.of(new ResourceEstimatesRange(
+                        Optional.of(new Range<>(
+                                Optional.empty(),
+                                Optional.of(Duration.valueOf("5m")))),
+                        Optional.empty(),
+                        Optional.of(new Range<>(
+                                Optional.empty(),
+                                Optional.of(DataSize.valueOf("500MB")))))),
+                Optional.empty(),
+                new ResourceGroupIdTemplate("global.foo"));
+
+        assertEquals(
+                smallQuerySelector.match(
+                        newSelectionCritera(
+                                "userA",
+                                null,
+                                ImmutableSet.of("tag1", "tag2"),
+                                new ResourceEstimates(
+                                        Optional.of(Duration.valueOf("4m")),
+                                        Optional.empty(),
+                                        Optional.of(DataSize.valueOf("400MB")))))
+                        .map(SelectionContext::getResourceGroupId),
+                Optional.of(resourceGroupId));
+
+        assertEquals(
+                smallQuerySelector.match(
+                        newSelectionCritera(
+                                "A.user",
+                                "a source b",
+                                ImmutableSet.of("tag1"),
+                                new ResourceEstimates(
+                                        Optional.of(Duration.valueOf("4m")),
+                                        Optional.empty(),
+                                        Optional.of(DataSize.valueOf("600MB")))))
+                        .map(SelectionContext::getResourceGroupId),
+                Optional.empty());
+
+        assertEquals(
+                smallQuerySelector.match(
+                        newSelectionCritera(
+                                "userB",
+                                "source",
+                                ImmutableSet.of(),
+                                new ResourceEstimates(
+                                        Optional.of(Duration.valueOf("4m")),
+                                        Optional.empty(),
+                                        Optional.empty())))
+                        .map(SelectionContext::getResourceGroupId),
+                Optional.empty());
+
+        StaticSelector hugeMemQuerySelector = new StaticSelector(
```

Comment:
Since we named the other variable as `smallQuerySelector` we can name this variable as `largeQuerySelector`.